doc/ref_model/chapters/chapter08.rst

```diff
@@ -210,14 +209,133 @@ The API and Cloud Brokerage models are illustrated in the following diagrams:
+
+.. list-table::
+   :widths: 16, 30, 30
+   :header-rows: 1
+   
+* - Ref
+  - Requirement
+  - Definition/Note
+* - hem.gen.001
+  - HEMP shall use only published APIs in managing component clouds. 
+  - For example, to accomplish the example in `hem.gen.003` it will use the published APIs of the target cloud.
+* - hem.gen.002
+  - HEMP shall publish all of the APIs used by any of its components.
+  - For example, the provided GUI portal shall only use HEMP published APIs.
+* - hem.gen.003
+  - HEMP shall provide for a common mechanism for interaction with its constituent clouds.
+  - For example, "create Account" across the different public clouds. 
+* - hem.gen.004
+  - HEMP shall generalise and define a common set of resources available to be managed in constituent clouds.
+  - Example resources: hosts (including BareMetal), Virtual Machines (VM), vCPU, Memory, Storage, Network, kubernetes clusters, kubernetes nodes, Images (OS, and others), credentials. For private cloud additional example resources: Racks, ToR/CE switches, Platform images. 
+
+Table : Hybrid, Edge, and Multi cloud operator Platform (HEMP) General Requirements
+
+.. list-table::
+   :widths: 16, 30, 30
+   :header-rows: 1
+   
+* - Ref
+  - Requirement
+  - Definition/Note
+* - hem.ops.001
+  - HEMP shall generalise and define a common set of management operations available in constituent clouds. Some operations may only be available for a subset of resources.
+  - For example, create, deploy, configure, start, suspend, stop, resume, reboot, delete, scale, list.
+* - hem.ops.002
+  - HEMP shall centrally manage all resources (across all constituent clouds).
+  - 
+* - hem.ops.003
+  - HEMP shall centrally operate all constituent clouds.
+  -
+* - hem.ops.004
+  - HEMP shall provide mechanisms to integrate new clouds.
+  - This may require pre-development of necessary capabilities for the support of HEMP abstractions, and impementation of connectivity with the new cloud.
+* - hem.ops.005
+  - HEMP shall provide mechanisms to drop a constituent cloud.
+  - This will result in deletion of all resources in the constituent cloud.
+* - hem.ops.006
+  - HEMP shall provide mechanisms and processes to onboard existing assets (resources, connectivity, etc.).
+  -
+* - hem.ops.007
+  - HEMP shall provide mechanisms and processes for the automated configuration management of all environments and resources.
+  -
+
+Table : Hybrid, Edge, and Multi cloud operator Platform (HEMP) Operability Requirements
+
+.. list-table::
+   :widths: 16, 30, 30
+   :header-rows: 1
+   
+* - Ref
+  - Requirement
+  - Definition/Note
+* - hem.lcm.001
+  - HEMP shall provide visibility into the health of all assets.
+  - 
+* - hem.lcm.002
+  - HEMP shall monitor all environments and assets.
```

Comment:
The requirement HEM.LCM.002 is the umbrella requirement of all requirements in this table. It can be include in all the other requirements.

Reply:
Yes, I wonder if HEM.LCM.002 is useful as the other requirements are addressing dedicated aspects of monitoring

Reply:
@karinesevilla we can delete if you feel strongly about it

Reply:
@pankaj Instead of deleting it, I would have placed it at first, switching  hem.lcm.001 and hem.lcm.002